### DISCUSSION TITLE
[notifications] Allow the default filter rule to be set for all repositories

### DISCUSSION THREAD

**Discussion by workgroupengineering** [community/community#7152] [Other Feature Feedback, Questions, & Ideas]
## Current behavior

- In the notification center, when you click on a repository all messages are displayed.

## Desiderated behavior

- In the notification center, add the option to show only unread notifications without setting a custom filter rule for each repository.

**Comment by PrplHaz4** [comment]
This would be a great option - my first click for every notification center visit and every repo under it is "unread".
